### PR TITLE
Fix/v3 bugs

### PR DESCRIPTION
**Pull request by MD-REHMAN**

### CODE REVIEW

File: src/components/composites/NumberInput/NumberInputStepper.tsx [hunk 1]
```diff
@@ -32,6 +32,13 @@ export const NBStepper = ({ children, ...props }: any) => {
         {...(disablitityCheck || isDisabled ? _disabled : {})}
         borderColor="transparent"
         style={style}
+        opacity={disablitityCheck || isDisabled ? 0.4 : 1}
+        {...(Platform.OS === 'web'
+          ? {
+              disabled: disablitityCheck || isDisabled,
```
Comment (theankurkedia) [inline src/components/composites/NumberInput/NumberInputStepper.tsx:38]: We have already added it in line 32. Why are we adding it again?

File: src/components/primitives/Input/index.tsx [hunk 2]
```diff
@@ -157,6 +157,12 @@ const Input = (
           editable={isDisabled || isReadOnly ? false : true}
           // borderRadius={50} //Remove variant props from StyledInput
           borderWidth={undefined}
+          {...(Platform.OS === 'web'
+            ? {
+                disabled: isDisabled,
```
Comment (theankurkedia) [inline src/components/primitives/Input/index.tsx:162]: Would this only be for web?

File: src/components/primitives/Radio/Radio.tsx [hunk 3]
```diff
@@ -41,6 +41,13 @@ const Radio = ({ icon, children, ...props }: IRadioProps, ref: any) => {
         justifyContent="center"
         alignItems="center"
         {...newProps}
+        opacity={isDisabled ? 0.4 : 1}
+        {...(Platform.OS === 'web'
+          ? {
+              disabled: isDisabled,
```
Comment (theankurkedia) [inline src/components/primitives/Radio/Radio.tsx:47]: Would this only be for web?

**Comment by MD-REHMAN** [comment]
- Updated Slider example: https://trello.com/c/sKjXLf1W
- Collapse fix: https://trello.com/c/Okc4f6ET
- Web accessibility and disable form elements view: https://trello.com/c/m78CW3mx & https://trello.com/c/VDo45BdZ

File: src/components/primitives/Slider/Slider.tsx [hunk 4]
```diff
@@ -187,6 +188,13 @@ class NBSlider extends React.Component<
             minWidth="100%"
             {...this.props}
             onLayout={this.onBarLayout}
+            opacity={this.props.isDisabled ? 0.4 : 1}
+            {...(Platform.OS === 'web' && this.props.isDisabled
+              ? {
+                  disabled: this.props.isDisabled,
```
Comment (theankurkedia) [inline src/components/primitives/Slider/Slider.tsx:194]: Again same for web?

File: src/components/primitives/Radio/Radio.tsx (hunk 3 above)
Comment (MD-REHMAN) [reply src/components/primitives/Radio/Radio.tsx:47]: It's only for accessibility reason to let the browser know that the div is disabled, as the cursor will be changing.

File: src/components/composites/NumberInput/NumberInputStepper.tsx (hunk 1 above)
Comment (MD-REHMAN) [reply src/components/composites/NumberInput/NumberInputStepper.tsx:38]: Adding disable on make the component unfit for `focus`. That's why adding disabled only for the web.

File: src/components/primitives/Slider/Slider.tsx (hunk 4 above)
Comment (MD-REHMAN) [reply src/components/primitives/Slider/Slider.tsx:194]: Same reason.

File: src/components/primitives/Input/index.tsx (hunk 2 above)
Comment (MD-REHMAN) [reply src/components/primitives/Input/index.tsx:162]: Adding disable on make the component unfit for focus. That's why adding disabled only for the web.

**Pull request merged**